src/hotspot/share/ci/ciMethodData.hpp

```diff
@@ -511,16 +511,16 @@ class ciMethodData : public ciMetadata {
   ciProfileData* first_data() { return data_at(first_di()); }
   ciProfileData* next_data(ciProfileData* current);
   DataLayout* next_data_layout(DataLayout* current);
-  bool is_valid(ciProfileData* current) { return current != NULL; }
-  bool is_valid(DataLayout* current)    { return current != NULL; }
+  bool is_valid(ciProfileData* current) { return current != nullptr; }
+  bool is_valid(DataLayout* current)    { return current != nullptr; }
 
   DataLayout* extra_data_base() const  { return data_layout_at(data_size()); }
   DataLayout* args_data_limit() const  { return data_layout_at(data_size() + extra_data_size() -
                                                                parameters_size()); }
 
-  // Get the data at an arbitrary bci, or NULL if there is none. If m
-  // is not NULL look for a SpeculativeTrapData if any first.
-  ciProfileData* bci_to_data(int bci, ciMethod* m = NULL);
+  // Get the data at an arbitrary bci, or null if there is none. If m
+  // is not nullptr look for a SpeculativeTrapData if any first.
+  ciProfileData* bci_to_data(int bci, ciMethod* m = nullptr);
```

Comment:
Again mixture of `null` and `nullptr` in comment. Even so the comment for `ciMethodData::bci_to_data()` in .cpp file says `nullptr`